cmd/cmd_image.go

```diff
@@ -661,8 +661,7 @@ func getLocalImageReader(flags *pflag.FlagSet, args []string) *fs.Reader {
 	imageName := args[0]
 	imagePath := path.Join(api.LocalImageDir, imageName)
 	if _, err := os.Stat(imagePath); err != nil {
-		if os.IsNotExist(err) && !strings.HasSuffix(imagePath, ".img") {
-			imagePath += ".img"
+		if os.IsNotExist(err) {
 			_, err = os.Stat(imagePath)
```

Comment:
This `os.Stat(imagePath)` call was there to check for the existence of an image file after adding the .img suffix; if we don't use the suffix anymore, we could avoid this call altogether, since it's redundant with the previous one at line 663.